Services/LDAP/classes/class.ilLDAPAttributeMapping.php

```diff
@@ -129,7 +129,7 @@ private static function getMappedFields(array $rules) : array
     {
         $fields = [];
         foreach ($rules as $rule) {
-            if (!strlen($rule['value'])) {
+            if ($rule['value'] === '') {
```

Comment:
According to the database schema `value` could be `null`. So you should also check against `null` if you use `===` here.